pkgs/tools/system/plan9port/builder.sh

```diff
@@ -8,13 +8,20 @@ plan9portLinkFlags()
     local -a linkFlags=()
     eval set -- "$NIX_LDFLAGS"
     while (( $# > 0 )); do
-        if [[ $1 = -rpath ]]; then
-            linkFlags+=( "-Wl,-rpath,$2" )
+        case "$1" in
+        -rpath|-macosx_version_min|-sdk_version)
+            linkFlags+=( "-Wl,$1,$2" )
             shift 2
-        else
+            ;;
+        -no_uuid)
+            linkFlags+=( "-Wl,$1" )
+            shift
+            ;;
+        *)
             linkFlags+=( "$1" )
```

Comment:
All the `LDFLAGS` should be sent to the linker instead just handling a few special cases.

Reply:
That works and makes it simpler.

Reply:
Great, if that works I can't think of any reason not to do it this way.